src/components/BlockTypeList.vue

```diff
@@ -10,11 +10,9 @@
       <template v-for="blockType in filteredBlockTypes" :key="blockType.id">
         <BlockTypeCardPreview :block-type="blockType">
           <template #actions>
-            <p-link :to="routes.blockCreate(blockType.slug)">
-              <p-button icon-append="PlusIcon" class="block-type-list__add">
-                Add
-              </p-button>
-            </p-link>
+            <p-button icon-append="PlusIcon" class="block-type-list__add" @click="handleAdd(blockType)">
```

Comment:
I don't want to lose the ability to copy links or open links in a new tab. Can you conditionally bind the route to the buttons `to` attribute? That way when this isn't using emit you still get a link and not just a button.

Reply:
It seemed simpler to add conditionality to the button here - let me know if you'd rather we keep handleOpen and bind to the `to` attribute

Reply:
Eh 3 days later I feel less strongly about this lol. Proceed.